clients/fossid-webapp/src/main/kotlin/Extensions.kt

```diff
@@ -22,18 +22,23 @@
 package org.ossreviewtoolkit.clients.fossid
 
 import java.io.File
+import java.util.Base64
```

Comment:
Note to myself: With Kotlin 1.8.20 we could use the [experimental Kotlin-native Base64 implementation](https://kotlinlang.org/docs/whatsnew-eap.html#experimental-support-for-base64-encoding-in-standard-library).